README.rst

```diff
@@ -1,12 +1,33 @@
 
-*******
-Cantera
-*******
+.. Cantera
 
-Version 2.4.0b1 (development)
+.. image:: http://cantera.github.io/docs/sphinx/html/_static/cantera-logo.png
+   :target: http://cantera.github.io/docs/sphinx/html/index.html#
+   :alt: cantera logo
+   :width: 675px
+   :align: center
+   
 
-.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.170284.svg
+.. |doi| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.170284.svg
    :target: https://doi.org/10.5281/zenodo.1174508
+   
+.. |codecov| image:: https://img.shields.io/codecov/c/github/Cantera/cantera/master.svg
+   :target: https://codecov.io/gh/Cantera/cantera?branch=master
+
+.. |travisci| image:: https://travis-ci.org/Cantera/cantera.svg?branch=master
+    :target: https://travis-ci.org/Cantera/cantera
+
+.. |appveyor| image:: https://ci.appveyor.com/api/projects/status/auhd35qn9cdmkpoj?svg=true
+    :target: https://ci.appveyor.com/project/Cantera/cantera
+
+.. |release| image:: https://img.shields.io/github/release/cantera/cantera.svg   
+   :alt: GitHub release
+
+.. |commits| image:: https://img.shields.io/github/commits-since/Cantera/cantera/latest.svg   
+   :alt: Github commits (since latest release)
+
+|doi| |codecov| |travisci| |appveyor| |release| |commits|
```

Comment:
My other concern with this is that it makes this file much less readable anywhere except on the Github landing page. I like the fact that the current version has a minimal amount of `rst` cruft at the top before you get to human-readable content.